### PR TITLE
Update to policy text in SECURITY.md

### PR DESCRIPTION
**Pull request by vladimirvivien**
**What this PR does / why we need it**:

This PR updates the text in the SECURITY.md policy file.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+
+
+
+## Mailing lists
+
+
+
+*   Use security@vmware.com to report security concerns to the VMware Security Team, who uses the list to privately discuss security issues and fixes prior to disclosure.
+*   Join the [Sonobuoy Distributors](https://groups.google.com/g/projectsonobuoy-distributors) mailing list for early private information and vulnerability disclosure. Early disclosure may include mitigating steps and additional information on security patch releases. See below for information on how Sonobuoy distributors or vendors can apply to join this list.
```
Comment (wilsonehusin) [inline SECURITY.md:87]: is this link correct?

Comment (vladimirvivien) [reply SECURITY.md:87]: @wilsonehusin the Sonobuoy Distributors email list is correct. Did you get an error ?

Comment (wilsonehusin) [reply SECURITY.md:87]: 🤔 maybe I'm not in it yet lol

Comment (vladimirvivien) [reply SECURITY.md:87]: @wilsonehusin Jonas added you. You should be.

**Review by wilsonehusin — APPROVED** [review]

**Pull request merged**